test/auth_test/auth_test.go

```diff
@@ -47,7 +47,7 @@ var _ = Describe("Service Manager Authentication", func() {
 	)
 
 	BeforeSuite(func() {
-		ctx = common.NewTestContext(nil)
+		ctx = common.NewTestContextBuilder().Build()
```

Comment:
Can these be simplified as `common.NewDefaultTestContext()` that calls `builder.build`?